packages/expo-cli/src/commands/client/index.js

```diff
@@ -19,15 +23,39 @@ const { IOS } = PLATFORMS;
 
 export default program => {
   program
-    .command('client:ios')
+    .command('client:ios [project-dir]')
     .option(
       '--apple-id <login>',
       'Apple ID username (please also set the Apple ID password as EXPO_APPLE_PASSWORD environment variable).'
     )
     .description(
       'Build a custom version of the Expo Client for iOS using your own Apple credentials and install it on your mobile device using Safari.'
     )
-    .asyncAction(async options => {
+    .asyncActionProjectDir(async (projectDir, options) => {
+      const servicesDisabled = {
+        push_notifications:
+          'not yet available until API tokens are supported for the Push Notification system',
+      };
+
+      // get custom project manifest if it exists
+      // Note: this is some random user's project, NOT the expo client manifest
+      const spinner = ora(`Finding a valid app.json..`).start();
+      const appJsonPath = options.config || path.join(projectDir, 'app.json');
+      const appJsonExists = await ConfigUtils.fileExistsAsync(appJsonPath);
+      const { exp } = appJsonExists ? await ConfigUtils.readConfigJsonAsync(projectDir) : {};
+
+      if (exp) {
+        spinner.succeed(`Found app.json at ${appJsonPath}`);
+      } else {
+        spinner.warn(`Unable to find app.json.`);
+      }
+      if (!idx(exp, _ => _.ios.config.googleMapsApiKey)) {
```

Comment:
```suggestion
      if (!has(exp, _ => _.ios.config.googleMapsApiKey)) {
```

Also, remember to import `has` from lodash above: `import has from 'lodash/has';`